docs/src/content/docs/guides/make-onetime-payment.mdx

```diff
@@ -73,8 +73,8 @@ Create an `incomingPayment` on the recipient wallet's resource server using the
 Add the `incomingAmount` object and define the following properties:
 
 - `value` : the maximum allowable amount that will be paid to the recipient's wallet address.
-- `assetCode` : the ISO 4217 currency code representing the underlying asset used to make the payment.
-- `assetScale` : the scale of amounts denoted in the corresponding asset code.
+- `assetCode` : the code representing the underlying asset used to make the payment. An ISO 4217 currency code should be used wheverever possible. The ISO 4217 currency code for the US Dollar is represented as USD.
+- `assetScale` : the number of decimal places used to represent the minimum value in the corresponding asset code. USD has an asset scale of 2 with a minimum value of $0.01. An integer amount of 1000 with an `assetCode` of USD and `assetScale` of 2 translates to $10.00
```

Comment:
Thoughts on:


```suggestion
- `assetScale` : the number of decimal places that defines the scale of the smallest divisible unit for the given asset code. It determines how an integer amount is scaled to derive the actual monetary value. For example, USD has an asset scale of 2 with the smallest unit being 0.01. An integer amount of `1000` with an `assetCode` of USD and `assetScale` of 2 translates to $10.00.
```

Reply:
On the other hand, I can see that we reuse the asset scale explanation in multiple places. Would it be easier to link to glossary directly instead of having 3 instances of the same explanation throughout the docs?

Reply:
I'm making updates in a different PR that actually remove the definitions for these terms from the guides specifically. I didn't mark it here because I didn't want to potentially cause merge conflicts later.